spyder/plugins/application/container.py

```diff
@@ -284,73 +279,99 @@ def _check_updates_ready(self):
 
         # Adjust the checkbox depending on the stored configuration
         option = 'check_updates_on_startup'
-        check_updates = self.get_conf(option)
-        box.set_checked(check_updates)
+        box.set_checked(self.get_conf(option))
+
+        header = _(f"<b>Spyder {latest_release} is available!</b> "
+                   f"<i>(you have {__version__})</i><br><br>")
```

Comment:
f-strings can't be used in strings meant for translation. So, please rewrite this (and any other string that uses f-strings) to use `format` instead.

Reply:
No problem; I'll do that. Why can't f-strings be used? What is the difference between f-string and format for the translation?

Reply:
I think the `gettext` package (the one we use to extract strings for translation) doesn't support f-strings. So, we're forced to use `format` for them.